src/main.rs

```diff
@@ -0,0 +1,79 @@
+extern crate systemstat;
+use chrono::DateTime;
+use chrono::TimeZone;
+use chrono::Utc;
+use socketcan::*;
+use std::io::Write;
+use std::os::unix::net::UnixStream;
+use std::process::Command;
+use std::sync::mpsc::channel;
+use std::thread;
+mod data;
+mod decode_data;
+mod master_mapping;
+mod message;
+
+fn main() {
+    let mut down_command = Command::new("sudo")
```

Comment:
Why do these all need to be mutable?

Reply:
im not entirely sure, I think adding args is mutating it